framework/web/UrlManager.php

```diff
@@ -414,6 +415,7 @@ public function createUrl($params)
                 $route .= '?' . $query;
             }
 
+            $route = ltrim($route, '/');
```

Comment:
this fixes the case where route is empty and a `/` suffix is used. see the test case added below (line 447).